tests-e2e/ocw-ci-test-course/video-tabs.spec.ts

```diff
@@ -2,6 +2,26 @@ import { test, expect } from "@playwright/test"
 import { CoursePage } from "../util"
 import { VideoElement } from "../util/VideoElement"
 
+test("Verify that the Download Button works for multiple embed videos in a page", async ({
+  page
+}) => {
+  const coursePage = new CoursePage(page, "course")
+  await coursePage.goto("pages/multiple-videos-series-overview/")
+  const videoElementsCount = await new VideoElement(page).count()
+  for (let i = 0; i < videoElementsCount; i++) {
```

Comment:
**Requested change:** Either add an assertion here:

```
expect(videoElementsCount).toBe(3)
```

or loop over a list with a definite length.

See my comment here: https://github.com/mitodl/ocw-hugo-themes/pull/1149#discussion_r1202469710

In general, I like to avoid iterating over dynamic lists in tests. If you need to, make an assertion to ensure that the list has the length you expect (and in particular that it does not have zero length).

Reply:
Done